core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java

```diff
@@ -1589,7 +1589,7 @@ public boolean downloadIfNecessary() {
                     return true;
                 }
                 String downloadURL = String.format(
-                        "https://github.com/QuiltMC/quiltflower/releases/download/%s/quiltflower-%s.jar",
+                        "https://repo.maven.apache.org/maven2/org/vineflower/vineflower/%s/vineflower-%s.jar",
```

Comment:
Can we somehow use the Maven resolver APIs here? I fear that this may not work in environments with a firewall, but maybe it can be revisited in a future PR

Reply:
I am not exactly sure what that means as I have never used it. I wouldn't mind looking into it in another PR to have less risks in that one. For my comprehension, would it be about fetching the jar during build so it can be used form the MAVEN_HOME when needed? Or like currently download as needed via something like shrinkwrap/resolver (this does not seem to help with firewall though)?

Reply:
I would argue this is exactly similar to what it used to be. So let's not make @manofthepeace 's life harder than it needs to be.